bedrock/firefox/urls.py

```diff
@@ -81,7 +81,6 @@
         ),
         name="firefox.mobile.get-app",
     ),
-    path("firefox/send-to-device-post/", views.send_to_device_ajax, name="firefox.send-to-device-post"),
```

Comment:
Note to self: what should we do here, just let the URL 404?

Reply:
@pmac interested in your opinion - what should we do with this URL, is it OK to just let this 404?

Reply:
My opinion is yes, it should be fine for this to 404. We try not to do that with pages people may have visited, but an API endpoint like this that was just for our pages, let's just kill it. 

The `/newsletter/` endpoint may be another story as I know that there are other forms on other Mozilla sites that post there directly. We'll have to deal with that later and just leave it in place for now.